zokrates_cli/tests/integration.rs

```diff
@@ -17,23 +17,17 @@ mod integration {
     #[ignore]
     fn test_compile_and_witness_dir() {
         let dir = Path::new("./tests/code");
-        if dir.is_dir() {
-            for entry in fs::read_dir(dir).unwrap() {
-                let entry = entry.unwrap();
-                let path = entry.path();
-                if path.extension().unwrap() == "witness" {
-                    let program_name =
-                        Path::new(Path::new(path.file_stem().unwrap()).file_stem().unwrap());
-                    let prog = dir.join(program_name).with_extension("code");
-                    let witness = dir.join(program_name).with_extension("expected.witness");
-                    let args = dir.join(program_name).with_extension("arguments.json");
-                    test_compile_and_witness(
-                        program_name.to_str().unwrap(),
-                        &prog,
-                        &args,
-                        &witness,
-                    );
-                }
+        assert!(dir.is_dir());
```

Comment:
Specifying a path not pointing to a directory no longer shortcuts the test execution